JavaScript/Topics/7.Promises/README.md

```diff
@@ -0,0 +1,6 @@
+# Promises In Javascript
+
+### Video Lecture On Promises
+Video-[promises](https://youtu.be/ASfaBnNbeDc)
+ 
+Made with Love By-Devyank
```

Comment:
It should be like this:
`Contributor: [Devyank Nagpal](https://github.com/2devyank)`

Reply:
Okay